### PR TITLE
Add Viewport-Hight Client Hint

### PR DESCRIPTION
**Pull request by domfarolino**
This is a companion PR to https://github.com/WICG/client-hints-infrastructure/pull/44, where we actually add the Viewport-Height CH and its associated spec.

Related:
 - https://github.com/WICG/responsive-image-client-hints/issues/6
 - https://github.com/WICG/responsive-image-client-hints/pull/7

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -158,6 +158,43 @@ In the absence of any other client hints, the server does the best it can and se
...
+-------
+
+The <code><dfn http-header export>Sec-CH-Viewport-Height</dfn></code> request header field gives a server information about the user-agent's current <a href="https://www.w3.org/TR/CSS2/visuren.html#viewport">viewport</a> height. It is a [=Structured Header=] whose value MUST be an [=structured header/integer=] greater than or equal to 0.
+
+If the value is zero, it indicates that there is no <a href="https://www.w3.org/TR/CSS2/visuren.html#viewport">viewport</a>.
+
+For [=fetches=] within web contexts, its value SHOULD be the {{Window}}'s current {{Window/innerHeight}}.
+
+Note: window.innerHeight includes horizontal scrollbar width. Do we want to use the width of the initial containing block (minus any scrollbar width>; gettable with <code>
+document.documentElement.clientHeight</code>), instead?
```
Comment (domfarolino) [inline index.bs:171]: This note is basically copying the one for Viewport-Width, however I'm not sure if either are accurate? I test around a bit on some pages with custom scollbars and couldn't find a discrepancy between the two values. Could someone provide a good example?

Comment (eeeps) [reply index.bs:171]: I was going by https://drafts.csswg.org/cssom-view/#dom-window-innerheight

Comment (domfarolino) [reply index.bs:171]: OK, then that seems right

**Comment by eeeps** [comment]
LGTM

**Pull request merged**